Proposal-eng.md

```diff
@@ -0,0 +1,64 @@
+# Proposal - UIC HCC Computer Society
+
+Dear Sir / Madam:
+
+
+
+We are computer technology enthusiasts in UIC from different majors and different grades, who founded UIC HCC Computer Society (Unofficial) in October, 2016, in hope of spreading the knowledge of computer science and technology, conveying the sprit of open source, and sharing as well as focusing on the frontier development of computer science. Today, we hope to establish UIC HCC Computer Society officially attaching to Innovation Center, which will be put in services soon since the UIC new campus have been moved smoothly. We hope to make contributions to students and staffs in UIC as well as the Industry-University-Research Integration plan of UIC with the support of this platform.
+
+This proposal will introduce the overview, vision, missions, tasks and expected achievements of UIC HCC Computer Society (hereafter abbreviated as Society). Simutaneously, we will list our expectations to Innovation Center and UIC.
+
+
+
+## Society Overview
+
+UIC HCC Computer Society established unofficially in October, 2016, and have 20 members from different majors and different grades (members list in appendix). We make our goal **Bridging You And Technology** as our responsibility, and spread knowledge related to computer science to UIC teachers and students in different occasions actively. With the emerge of new technologies in computer science, we find it difficult to achieve our goal and keep our development under the unofficial operation mode. Therefore, our society wish to obtain the strong support from UIC and Innovation Center to carry out carry out more science-popularising lectures, practices, and research activities.
+
+Currently, we have our public website, GitHub Organization, mail system and WeChat official account.
+
+
+
+## Vision of Society
+
+In order to implement a fulfilling practice on the fundamental goal of Society, **Bridging You And Technology**, our society hopes to carry out **popularization of knowledge related to computer science**, **software and hardware development** cooperated with our school or society organizations, **participation in open source projects** and **research on frontier theories of computer science and their application**.
+
+### Popularizing knowledge related to computer science
+
+Though computer has gone into our life, for most people, computer is still a "black-box". Meanwhile, many concepts of computer science such as Blockchain, Artificial Intelligence and Big Data are very popular. New products and concepts in computer science field emerge one after the other. Many people are interested in these "high technology" products and concepts.
```

Comment:
1. Computer has been very common in our life and many concepts of computer science namely Blockchain.....are very popular in the field but not yet to be known by ordinary people that is interested in it, for which it is still like a "black-box".

Reply:
Please commit and push.